PlayKitBroadpeak/Classes/BroadpeakMediaEntryInterceptor.swift

```diff
@@ -59,23 +67,24 @@ import KalturaPlayer
             return
         }
         
-        self.config = config
-        
-        streamingSession.stop()
-        SmartLib.stopStreamingSession()
-        SmartLib.release()
-        
-        SmartLib.initSmartLib(self.config.analyticsAddress,
-                              nanoCDNHost: self.config.nanoCDNHost,
-                              broadpeakDomainNames: self.config.broadpeakDomainNames)
-        
-        streamingSession = SmartLib.createStreamingSession()
+        if !self.config.isEqual(config) {
```

Comment:
not sure this code is fully functional 
since the session is opened by apply and this code will be called after wards  who will call the start